.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
this should also be updated to allow running this check on PRs against all branches